### PR TITLE
Working Backdrop conversion of 7.x-1.26.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop-contrib/eu_cookie_compliance/issues/3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,60 +10,93 @@ data while a transaction is in progress, the user must be informed and
...
-- Marcin Pajdzik (marcin-pajdzik)
-- Achton Smidt Winther (achton)
-- mibfire
+- Graham Oliver (https://github.com/Graham-72).
+- Seeking additional maintainers.
+
+Credits
+-------
+
+- Ported to Backdrop CMS by Betty Boop (https://github.com/username).
```
Comment (klonos) [inline README.md]:
```suggestion
- Ported to Backdrop CMS by [Graham Oliver](https://github.com/Graham-72)
```

File: eu_cookie_compliance.install [hunk 2]
```diff
@@ -1,38 +1,420 @@
...
+    $eu_cookie_settings['withdraw_enabled'] = 0;
+  }
+  variable_set('eu_cookie_compliance', $eu_cookie_settings);
 }
 
 /**
- * Move settings from variables to config.
+ * Ensure that all "eu_cookie_compliance" parameters have a default value.
  */
-function eu_cookie_compliance_update_1000()
```
Comment (klonos) [inline eu_cookie_compliance.install:30]: Should this have been removed?

File: config/eu_cookie_compliance.settings.json [hunk 3]
```diff
@@ -1,34 +1,71 @@
...
-    "popup_find_more_button_message": "More info",
-    "popup_hide_button_message": "Hide",
-    "popup_agreed": {
-      "value": "<h2>Thank you for accepting cookies</h2> <p>You can now hide this message or find out more about cookies.</p>",
-      "format": "full_html"
+    "_config_name": "eu_cookie_compliance.settings",
+    "_module": "eu_cookie_compliance",
+    "eu_cookie_compliance_domain": "",
+    "eu_cookie_compliance_cookie_lifetime": "100",
+    "eu_cookie_compliance": {
```
Comment (klonos) [inline config/eu_cookie_compliance.settings.json:6]: It seems that this has previously been `eu_cookie_compliance_en` so an update hook might be needed.

Comment (klonos) [reply config/eu_cookie_compliance.settings.json:6]: ...yep. https://backdropcms.org/project/usage/eu-cookie-compliance shows about a dozen sites using it.

File: README.md (hunk 1 above)
Comment (jenlampton) [reply README.md]: hah! oops :) fixed.

File: eu_cookie_compliance.install (hunk 2 above)
Comment (jenlampton) [reply eu_cookie_compliance.install:30]: likely not...  okay. Update path fixed.

File: config/eu_cookie_compliance.settings.json (hunk 3 above)
Comment (jenlampton) [reply config/eu_cookie_compliance.settings.json:6]: yes, added one to bring the settings back in line with the 1.x branch `eu_cookie_compliance_update_1001()`.

**Comment by jenlampton** [comment]
closing (will file a new PR in a minute, with a less confusing branch name.

**Comment by jenlampton** [comment]
See https://github.com/backdrop-contrib/eu_cookie_compliance/pull/10

**Pull request closed without merging**